uberenv.py

```diff
@@ -814,9 +842,15 @@ def install(self):
                     print("[install complete!]")
         # otherwise we are in the "only dependencies" case and the host-config
         # file has to be copied from the do-be-deleted spack-build dir.
+        # -------
+        # NOTE: USING DEV BUILD, there is no `spack-build` dir
```

Comment:
@cyrush yes there is a `spack-build` dir, it is created as the source top dir.
The role of `dev-build` is to use the local version of the sources, that's why it is convenient: the host-config gets generated next to the local version of the source (in `src/spack-build`).
Then, uberenv moves the host-config file from `src/spack-build` to `src`, and removes `src/spack-build`.
If I understand correctly, that's where is becomes a problem for you: you may have several host-config files there.

Reply:
Correct. Overall, I want to avoid creating or deleting directories anywhere other than the prefix.

If things are limited to the prefix, I can avoid surprises.

Reply:
I though I checked to see if the `spack-build` was created for my case, and I didn't see evidence it happened (maybe b/c it was deleted before I could check) but I though I put in echos to see if ubrerenv saw it.  I can double check when I am back working on this.

Reply:
I removed this comment, not sure if its still an issue but it relates to a path I am not testing.